src/lfortran/semantics/ast_common_visitor.h

```diff
@@ -1327,6 +1327,35 @@ class CommonVisitor : public AST::BaseVisitor<Derived> {
             left, end_bin_op->m_op, right, end_bin_op->m_type, end_bin_op->m_value));
     }
 
+    bool dimensionAttribute_error_check(ASR::expr_t* dim_expr) {
```

Comment:
Use the following naming convention to be consistent with the rest of the code:
```suggestion
    bool dimension_attribute_error_check(ASR::expr_t* dim_expr) {
```